Knight: NavigationLayout

chineduknight opened the pull request:
-Added the Navbar to the ui
-Added the Sidebar to the ui
-Added the NavigationLayout to layout

[finished]

#### What does this PR achieve?
-Adds the Navbar to the ui
-Adds the Sidebar to the ui
-Adds the NavigationLayout to layout

#### Any background context?
#### Manual Testing Steps
#### Screenshots:
![ftSidebar](https://user-images.githubusercontent.com/50283514/77853099-38ebad00-71da-11ea-9654-276956f1fe8f.jpg)
![ft02header](https://user-images.githubusercontent.com/50283514/77853100-3ab57080-71da-11ea-9b3e-5982c0a0029a.jpg)
![ft01](https://user-images.githubusercontent.com/50283514/77853101-3b4e0700-71da-11ea-8c8c-bb2e4e7ee8d9.jpg)

#### Relevant Trello Tickets
- [Create Portal Layout Component](https://trello.com/c/iSDlQKNd/99-create-portal-layout-component)

chidioguejiofor commented:
@ChineduOguejiofor  There is a conflict in your packge.json. Please review.Also rename your Pr titile to have the format `YourName: Short Title`

chidioguejiofor commented on src/components/ui/OrganizationHeader/index.tsx:
```diff
@@ -0,0 +1,56 @@
+
+export default function OrganizationHeader() {
+  const sampleImage =
+    'https://images.unsplash.com/photo-1535713875002-d1d0cf377fde?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1000&q=80';
```
This should have actually been passed as a prop since we want it to change

chidioguejiofor replied:
You can use this url when creating a sample image though

chineduknight replied:
I have added the prop

chidioguejiofor reviewed (commented):
Looks good. Once you resolve the conflict this can be merged. Also made a single liner change.

Commit:
feat(NavigationLayout): <Navigation and sidebar>

-Added the Navbar to the ui
-Added the Sidebar to the ui
-Added the NavigationLayout to layout
-added props for Navbar

[finished]
